pipeline-model-definition/src/test/resources/buildPluginParentPOM.groovy

```diff
@@ -17,11 +17,11 @@ pipeline {
                         if (isUnix()) {
                             sh 'echo "M2_HOME: ${M2_HOME}"'
                             sh 'echo "JAVA_HOME: ${JAVA_HOME}"'
-                            sh 'mvn clean verify -Dmaven.test.failure.ignore=true'
+                            sh 'mvn clean verify -Dmaven.test.failure.ignore=true -e'
                         } else {
                             bat 'echo "M2_HOME: %M2_HOME%"'
                             bat 'echo "JAVA_HOME: %JAVA_HOME%"'
-                            bat 'mvn clean verify -Dmaven.test.failure.ignore=true'
+                            bat 'mvn clean verify -Dmaven.test.failure.ignore=true -e'
```

Comment:
Seems reasonable to have this in case of error.

Reply:
I was trying to track down a failure which did not in fact reappear.